src/ape_test/provider.py

```diff
@@ -254,3 +254,10 @@ def get_virtual_machine_error(self, exception: Exception, **kwargs) -> VirtualMa
 
         else:
             return VirtualMachineError(base_err=exception, **kwargs)
+
+    @raises_not_implemented
```

Comment:
Won't this still raise? I don't get the comment in the body

Reply:
it is implemented in the base class.
However, we have another handler somewhere that handles the equivalent error from `web3` and raises `APINotImplemented` at that point, as if it had this decorator.
So by adding this decorator, it is small perf by bypassing any call to web3 and just raising immediately, because we know it is not going to work for EthTester, unfortunately.

On the note, I have looked into what it would take a little bit to implement this kind of stuff in EthTester.. I think it needs some extra handling for irregular chain activity like this. At least that is what it seems like Hardhat does.

Reply:
If we were using hardhat or something, I could call `set_code()` instead of deploying a contract for some of these proxies

Reply:
`Web3Provider` implements `get_storage_at()` but it fails in `EthTester`, which maybe is surprising idk

Reply:
good news: https://github.com/ethereum/eth-tester/pull/264
so im going to delete this change so it just works when it comes, and it has no effect now anyways besides short-circuit